src/blocks/header.rs

```diff
@@ -86,40 +86,39 @@ impl RawBlockHeader {
         parent_epoch: ChainEpoch,
         prev_entry: &BeaconEntry,
     ) -> Result<(), Error> {
-        let (cb_epoch, curr_beacon) = b_schedule
+        let (_cb_epoch, curr_beacon) = b_schedule
             .beacon_for_epoch(self.epoch)
             .map_err(|e| Error::Validation(e.to_string()))?;
-        let (pb_epoch, _) = b_schedule
+        let (_pb_epoch, _) = b_schedule
             .beacon_for_epoch(parent_epoch)
             .map_err(|e| Error::Validation(e.to_string()))?;
 
-        if cb_epoch != pb_epoch {
```

Comment:
This aligns with https://github.com/filecoin-project/lotus/pull/11572/files#diff-587eaf0df6b60dbf741d19bbe39439f6f48ffe171e82a71c76b82484ba48f386R55